Dockerfile

```diff
@@ -146,7 +146,15 @@ RUN for postgisv in ${POSTGIS_VERSIONS}; do \
 # This need to be done after the PostgreSQL packages have been installed,
 # to ensure we have the preferred libpq installations etc.
 RUN apt-get install -y python3-etcd python3-requests python3-pystache python3-kubernetes python3-pysyncobj
-RUN apt-get install -y patroni
+RUN apt-get install -y patroni=2.1.3-\*
+# Patch Patroni code with changes from https://github.com/timescale/patroni/pull/1
+# NOTE: This is a temporary solution until changes land upstream.
+ARG TIMESCALE_STATIC_PRIMARY
+RUN if [ "${TIMESCALE_STATIC_PRIMARY}" != "" ]; then \
+    wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.1/patroni/ha.py > /usr/lib/python3/dist-packages/patroni/ha.py && \
+    wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.1/patroni/config.py > /usr/lib/python3/dist-packages/patroni/config.py && \
+    wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.1/patroni/validator.py > /usr/lib/python3/dist-packages/patroni/validator.py; \
+    fi
```

Comment:
Hey @feikesteenbergen && @alexeyklyukin just wanted to ping you guys here again.
- these bits are now optionally included via build-arg;
- a new build matrix stage has been added to the tagged CI pipeline, which will suffix the release tag with `-patroni-static-primary` so that we can easily differentiate the images;
- this will allow us to enable these changes throughout our stack, and easily rollback the Patroni changes with a minimal image change.